recipes/segemehl/meta.yaml

```diff
@@ -0,0 +1,33 @@
+package:
+  name: segemehl
+  version: 0.2.0
+
+about:
+  home: http://www.bioinf.uni-leipzig.de/Software/segemehl/
+  summary: Short read mapping with gaps
+
+build:
+  number: 0
+
+source:
+  fn: segemehl_0_2_0.tar.gz
+  url: "http://www.bioinf.uni-leipzig.de/Software/segemehl/segemehl_0_2_0.tar.gz"
+  sha256: 2575139d2f2ba7bd534afcbe485c0570c177d1faaeb267c4cd873e65b8e72afb
+
+requirements:
+  build: 
+    - llvm # [osx]
+    - gcc # [linux]
+    - libgcc # [linux]
+    - zlib
+    - ncurses
+  run: 
+    - libgcc # [linux]
+    - zlib
```

Comment:
As far as I understand the pins are defined in scripts/env_matrix.yml
and the ZLIB pin is not defined yet. Can we add
CONDA_ZLIB: "1.2.8"
?

On 05/14/2016 01:43 PM, Björn Grüning wrote:
> In recipes/segemehl/meta.yaml
> <https://github.com/bioconda/bioconda-recipes/pull/1553#discussion_r63275016>:
> 
>> +
>> +source:
>> +  fn: segemehl_0_2_0.tar.gz
>> +  url: "http://www.bioinf.uni-leipzig.de/Software/segemehl/segemehl_0_2_0.tar.gz"
>> +  sha256: 2575139d2f2ba7bd534afcbe485c0570c177d1faaeb267c4cd873e65b8e72afb
>> +
>> +requirements:
>> +  build: 
>> +    - llvm # [osx]
>> +    - gcc # [linux]
>> +    - libgcc # [linux]
>> +    - zlib
>> +    - ncurses
>> +  run: 
>> +    - libgcc # [linux]
>> +    - zlib
> 
> One new thing :-)
> We are pinning libraries like zlib and ncurses.
> Can you grep in other recipes how this looks like.
> 
>   * zlib {{CONDA_ZLIB}} or similar. Sorry in a train right now.
> 
> Great work. Now we can put the Galaxy wrapper under testing.
> 
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
> <https://github.com/bioconda/bioconda-recipes/pull/1553/files/2599d3c21e97d488b563991c9ec4dfe9522022c2#r63275016>
> 


-- 
Dr. Sebastian Will
Bioinformatics/IZBI
Härtelstrasse 16-18, Room 448.1
University Leipzig
D-04107 Leipzig

Tel.: +49 (0) 341 97 16689